packages/search-input/src/SearchResult/SearchResult.tsx

```diff
@@ -20,9 +16,7 @@ import {
 } from './SearchResult.styles';
 import { SearchResultProps } from './SearchResult.types';
 
-export const SearchResult = Polymorphic<
-  InferredPolymorphicProps<SearchResultProps>
->(
+export const SearchResult = InferredPolymorphic<SearchResultProps>(
   (
     {
       as = 'li' as PolymorphicAs,
```

Comment:
won't this break now?

Reply:
Won't "break", just wouldn't be exactly what we want 🙃 
Fixed